### PR TITLE
[climate] Migrate honeywell docs for climate-1.0

### PR DESCRIPTION
**Pull request by zxdavb**
**Description:**
Migrate honeywell docs for `climate-1.0`

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/9816"></a>

### CODE REVIEW

**Comment by klaasnicolaas** [comment]
This PR must be in the `climate-1` branch 😉

File: source/_components/sonos.markdown [hunk 1]
```diff
@@ -114,6 +114,17 @@ Night Sound and Speech Enhancement modes are only supported when playing from th
 | `night_sound` | yes | Boolean to control Night Sound mode.
 | `speech_enhance` | yes | Boolean to control Speech Enhancement mode.
 
+### Service `sonos.play_queue`
```
Comment (frenck) [inline source/_components/sonos.markdown]: This should not be here, please restore this part.
(Unrelated).

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
Add honeywell for climate 1.0 in #9799

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Lets not stall this and merge it into next, and make a mention in the main PR.

**Comment by frenck** [comment]
Thanks, @zxdavb!

**Comment by zxdavb** [comment]
Sorry - my git is pants.

**Pull request merged**